(self.webpackChunkblog=self.webpackChunkblog||[]).push([[5933],{35933:n=>{n.exports="// 完善的框架都有相应的错误捕获机制\nconst Koa = require('../application')\n\nconst app = new Koa()\n\nconst responseData = {}\n\n// 第四版测试: 错误机制测试\napp.use(async (ctx, next) => {\n  throw new Error('ooops')\n})\n\napp.on('error', (err) => {\n  console.log('111', err.stack)\n})\n\napp.listen(3000, '127.0.0.1', () => {\n  console.log('listening on the 3000')\n})"}}]);